src/models/histories.js

```diff
@@ -0,0 +1,62 @@
+module.exports = function() {
+  var db = require('./db.js');
+  var sequelize = db.connection;
+  var util = require('../../lib/util');
+
+// Create
+  function _create(payload, err, success) {
+    var cleanData = util.scrubData(payload);
+    db.histories.create(cleanData).then(success).catch(err);
+  }
+// Update
```

Comment:
Does not match Comment Formatting in the airbnb style guide. Also, all comments on this file are inconsistent.